bin/shocker

```diff
@@ -354,8 +355,8 @@ def main():
             # Executing the relaxation simulation
             # =================================================================
 
-            grompp = '{} grompp -f {} -c {} -p {} -o {} -n {}'
-            mdrun = '{} mdrun -s {} -v -x {} -c {}'
+            grompp = '{} grompp -f {} -c {} -p {} -o {} -n {} -maxwarn 1'
+            mdrun = '{} mdrun -ntmpi 6 -ntomp 3 -dlb auto -pin on -nb gpu -bonded gpu -s {} -v -x {} -c {}'
```

Comment:
This is specific to snellius and should be changed.

Reply:
This should not be in the top-level code anyways. We have to fix this.

Reply:
So they should be defined in a separate module?

Reply:
the user should be able to input them from an external data file I think. Otherwise you need to change the code depending on the architecture

Reply:
yes I agree with that